lighthouse-core/scripts/roll-to-devtools.sh

```diff
@@ -80,6 +80,12 @@ rsync -avh "$lh_webtests_dir" "$fe_webtests_dir" --exclude="OWNERS" --delete
 lh_webtests_exp_dir="third-party/chromium-webtests/webtests/platform/generic/http/tests/devtools/lighthouse/"
 fe_webtests_exp_dir="$dt_dir/test/webtests/platform/generic/http/tests/devtools/lighthouse"
 rsync -avh "$lh_webtests_exp_dir" "$fe_webtests_exp_dir" --exclude="OWNERS" --delete
+
+# copy e2e tests
+lh_e2e_dir="third-party/devtools-tests/e2e/lighthouse/"
+fe_e2e_dir="$dt_dir/test/e2e/lighthouse"
+rsync -avh "$lh_e2e_dir" "$fe_e2e_dir" --exclude="OWNERS" --exclude="BUILD.gn" --delete
```

Comment:
yes, this currently breaks build in CI. whatever. let's just wait for CL to land.